checkout-ui-custom/src/_js/_countries.js

```diff
@@ -1,4 +1,17 @@
 module.exports._countriesrules = {
+  FRA: {
+    city: {
+      types: ['locality'],
+      valueIn: 'long_name',
+    },
+    number: true,
+    postalCode: true,
```

Comment:
I believe this is no longer needed as these rules come back from the `address form` instead of this file